docs/en/observability/synthetics-private-location.asciidoc

```diff
@@ -1,5 +1,28 @@
 [[synthetics-private-location]]
-= Add a private location
+= Monitor resources on private networks
+
+To monitor resources on private networks you can either:
+
+* Allow the Synthetics service to access your private endpoints.
+* Use {agent} to create a Private Location.
+
+Private Locations via Elastic Agent require only outbound connections from your network, while allowing access to the Synthetics Service requires allowing
+inbound access, thus posing an additional risk that users must assess.
+
+[discrete]
+[[monitor-via-access-control]]
+= Allow the synthetics service access to your private network
+
+To give the Synthetics Service access to a private endpoint, use IP address filtering, HTTP authentication, or both.
+
+To grant access via IP, use https://manifest.synthetics.elastic-cloud.com/v1/ip-ranges.json[this list of egress IPs].
+The addresses and locations on this list may change, so automating updates to
+filtering rules is recommended. IP filtering alone will allow all users of our synthetics service access to your endpoints, if this
+is a concern consider adding additional protection via user/password authentication via a proxy like nginx.
```

Comment:
>The _proper_name is Elastic's global managed infrastructure like we use elsewhere, should probably use that for consistency

@paulb-elastic @andrewvc does "Elastic's global managed infrastructure" make sense in all these contexts? 

```suggestion
To monitor resources on private networks you can either:

* Allow Elastic's global managed infrastructure to access your private endpoints.
* Use {agent} to create a Private Location.

Private Locations via Elastic Agent require only outbound connections from your network, while allowing Elastic's global managed infrastructure to access a private endpoint requires
inbound access, thus posing an additional risk that users must assess.

[discrete]
[[monitor-via-access-control]]
= Allow access to your private network

To give Elastic's global managed infrastructure access to a private endpoint, use IP address filtering, HTTP authentication, or both.

To grant access via IP, use https://manifest.synthetics.elastic-cloud.com/v1/ip-ranges.json[this list of egress IPs].
The addresses and locations on this list may change, so automating updates to
filtering rules is recommended. IP filtering alone will allow all users of Elastic's global managed infrastructure access to your endpoints, if this
is a concern consider adding additional protection via user/password authentication via a proxy like nginx.
```

Reply:
> ...does "Elastic's global managed infrastructure" make sense in all these contexts?

I do think that it is wordy, but it still makes sense to me when reading it.  My worry is introducing other terms (we've had this in the past).  I know internally we sometimes distinguish them as _Public_ and _Private_ locations, but haven't used that externally as far as I can remember.

Reply:
>I do think that it is wordy, but it still makes sense to me when reading it. My worry is introducing other terms (we've had this in the past).

I agree. For the purposes of this PR I think it's ok to use the wordy phrase to keep docs consistent. Then we can revisit   this terminology separately, and if we decide to change to something like "Synthetics Service" we can make sure that change is applied across all docs. What do you think?

Reply:
I'm fine with the wordier phrase